modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -4217,6 +4217,32 @@ public static void close(@Nullable AutoCloseable rsrc, @Nullable IgniteLogger lo
         }
     }
 
+    /**
+     * Closes given socket logging possible checked exception.
+     *
+     * @param sock Socket to close. If it's {@code null} - it's no-op.
+     * @param log Logger to log possible checked exception with (optional).
+     */
+    public static void close(@Nullable Socket sock, @Nullable IgniteLogger log) {
+        if (sock != null) {
+            try {
+                //avoid tls 1.3 incompatibility https://bugs.openjdk.java.net/browse/JDK-8208526
```

Comment:
This comment is inconsistent with `// Avoid java 12 bug see https://bugs.openjdk.java.net/browse/JDK-8219658` in `closeQuiet` - maybe update both?